Maskbook/Scene/App/RedPacket/ViewModel/NftLuckyDropViewModel.swift

```diff
@@ -16,8 +17,8 @@ struct CollectiableGroup: Hashable {
 struct NftRecpacketDraft {
     let message: String
     let collectibles: [Collectible]
-    let gasFeeItem: GasFeeCellItem
     let token: Token
```

Comment:
I don't need token when creating nftRedpacket. 
If you don't need too, you can remove it.